src/components/RegisterPage/RegisterPage.jsx

```diff
@@ -0,0 +1,175 @@
+            <Card.Header className="card__header">
+                <Card.Title as="h3">Sign up</Card.Title>
+            </Card.Header>
+            <Card.Body>
+                <Form noValidate validated={validated} onSubmit={submitForm}>
+                    <Row>
+                        <Col>
+                            <Form.Group controlId="firstname">
+                                <Form.Label>First name:</Form.Label>
+                                <Form.Control
+                                    type="text"
+                                    required
+                                    placeholder="John"
+                                    value={data.name}
+                                    onChange={e => setData({...data, name: e.target.value})}
+                                />
+                                <Form.Control.Feedback type="invalid">
+                                    First name must be provided
+                                </Form.Control.Feedback>
+                            </Form.Group>
+                        </Col>
+                        <Col>
+                            <Form.Group controlId="lastname">
+                                <Form.Label>Last name:</Form.Label>
+                                <Form.Control
+                                    type="text"
+                                    placeholder="Doe"
+                                    required
+                                    value={data.surname}
+                                    onChange={e => setData({...data, surname: e.target.value})}
+                                />
+                                <Form.Control.Feedback type="invalid">
+                                    Last name must be provided
+                                </Form.Control.Feedback>
+                            </Form.Group>
+                        </Col>
+                    </Row>
+                    <Form.Group controlId="email">
+                        <Form.Label>Email:</Form.Label>
+                        <Form.Control
+                            type="email"
+                            required
+                            placeholder="johndoe@example.com"
+                            value={data.email}
+                            onChange={e => setData({...data, email: e.target.value})}
+                        />
+                        <Form.Control.Feedback type="invalid">
+                            Correct email must be provided
+                        </Form.Control.Feedback>
+                    </Form.Group>
+                    <Form.Group controlId="password">
+                        <Form.Label>Password:</Form.Label>
+                        <Form.Control
+                            type="password"
+                            minLength={8}
+                            required
+                            isInvalid={data.password && data.password_R && data.password !== data.password_R}
```

Comment:
Okeej, może to jest jakaś składnia, ale tego nie rozumiem xd w sensie nie wystarczy po prostu data.password !== data.password_R 
?